src/server.ls

```diff
@@ -3,7 +3,7 @@ require! <[ express fs path jade bluebird body-parser ./bundler livescript babel
 
 defaults =
   environment: process.env.NODE_ENV or 'development'
-  port: 3000
+  port: process.env.ARCH_PORT
```

Comment:
can we make it `process.env.ARCH_PORT || 3000`? In case you run the server without the CLI and don't set anything in your environment